test/clientconn_state_transition_test.go

```diff
@@ -498,3 +502,20 @@ func keepReading(conn net.Conn) {
 	for _, err := conn.Read(buf); err == nil; _, err = conn.Read(buf) {
 	}
 }
+
+// stayConnected makes cc stay connected by repeatedly calling cc.Connect()
+// until the state becomes Shutdown or until 10 seconds elapses.
```

Comment:
This comment no longer applies as the previous helper did, as the 10 seconds is no longer hardcoded. Switch to context expires.

Reply:
done, thanks